### DISCUSSION TITLE
Reference list per page, Quarto??

### DISCUSSION THREAD

**Discussion by cjsrussell** [quarto-dev/quarto-cli#7143] [Q&A] [answered]
### Description

As the title says! I have a quarto book/site project and would like per page references instead of one long reference list at the end. 

I have tried specifying the bibliography in the YAML for each page, and this works in terms of in-text citations,  but ` ::: {#refs} ::: `
only generates a reference list for the first page of the document, even if it is included per page.

Any way to do this?

**Comment by mcanouil** [comment, marked as answer]
Hi,

from <https://quarto.org/docs/extensions/> or <https://github.com/mcanouil/awesome-quarto#extensions>, there is an extension for that:
- https://github.com/pandoc-ext/section-bibliographies

![image](https://github.com/quarto-dev/quarto-cli/assets/8896044/821beb65-939f-4263-9394-8705895f74b2)

**Reply by cjsrussell** [reply]
Thanks for your quick response - I didn't see that.

Thanks!

**Reply by mcanouil** [reply]
You're welcome!
